src/app/_components/EventLinkModal.js

```diff
@@ -1,13 +1,55 @@
 import PropTypes from 'prop-types';
 import { QRCodeSVG } from 'qrcode.react';
+import Instagram from '@icons/instagram.svg';
+import Facebook from '@icons/facebook.svg';
 import { Modal } from '@components/Modal';
 import { PillButton } from '@components/PillButton';
 import { Heading, Paragraph } from '@components/Typography';
+import { useMediaQuery } from '@hooks';
```

Comment:
```js
import { useMediaQuery } from '@mui/material';
```
You could use out-of-the box solution from mui rather than create a custom hook.

Reply:
we should not use this, as 'mui' is not used in projcet directly, and only listed as peer dependency of react-admin we need to add to our own dependencies to be able to customise some react-admin elements

since our app do not use 'mui' officially, removing it from dependencies should not break client part, and this imports would violate this rule

Reply:
Am I missing something?

![image](https://github.com/keenethics/mayak/assets/103147058/e72a9004-781a-412b-a3e5-e8c1b49b29f1)

Reply:
yep

we have mui there because react-admin uses mui, and to create custom controls we need to import elements from mui ( its preferred way of making them according to react-admin docs ), and eslint does not want us to import transitive dependencies directly

not because we actually 'use' mui in our app components

so i would prefer separate hook instead of mui one in this particular case

so i would avoid using mui in our main app to prevent that dependency spreading out